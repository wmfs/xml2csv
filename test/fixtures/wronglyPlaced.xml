<Weird>lkezjflk
  <Case>weird text at wrong place !
    <First>OK 1</First>
    <Second>OK 2</Second>
  </Case>
</Weird>
