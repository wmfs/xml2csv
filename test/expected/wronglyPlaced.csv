first,second
"OK 1","OK 2"
